tests/e2e/LicenseCheckerCase.php

```diff
@@ -80,6 +80,40 @@ public function test_failure_with_invalid_allowed_licenses_file(): void
             );
     }
 
+    public function test_failure_with_invalid_allowed_licenses_file_containing_only_text(): void
+    {
+        $this->commandTester->execute([
+            '--allow-file' => 'tests/data/default/invalid_allowed_licenses.txt',
+            '--path' => $this->path,
+        ]);
+
+        CommandTesterAsserter::assertThat($this->commandTester)
+            ->doesNotContainInStdout('ShouldNotAppearInOutput')
```

Comment:
This assertion is an extra safety net, as we're not using a sub-process to run the command and it's done in the same PHP process as the tests it ends up being stdout of the main process rather than run through `symfony/console`'s `OutputInterface`. This means it will never actually be in the tested command's output at present even if it is written to screen.